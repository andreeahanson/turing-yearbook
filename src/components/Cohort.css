article {
  border: 1px solid black;
  width: 250px;
  margin-top: 20px;
  background: white;
}

img {
  height: 100px;
  width: 100px;
  border-radius: 50%;
} 