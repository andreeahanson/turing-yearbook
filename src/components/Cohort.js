import React from 'react';
import Person from './Person';
import './Cohort.css';

const Cohort = props => {
  return <section>
    <Person person={props.person}/>
  </section>

}

export default Cohort; 
