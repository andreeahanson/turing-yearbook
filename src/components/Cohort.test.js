import React from 'react';
import Cohort from './Cohort';
import { shallow, mount } from 'enzyme';

const mockData = [
  {id: 1, name: 'Eric', quote: 'You miss all the shots you don\'t take.', superlative: 'Most Likely to Win the PGA', photo: './eric.jpg'},
  {id: 2, name: 'Amy', quote: 'I believe in you, cuties!', superlative: 'Most Likely to Be an Actual Superhero', photo: './amy.jpg'}
]

describe('Person', () => {
  it('should match the snapshot with all data passed in correctly', () => {
    const wrapper = shallow(<Cohort
      staff = {mockData}
    />)

    expect(wrapper).toMatchSnapshot();
  })
})