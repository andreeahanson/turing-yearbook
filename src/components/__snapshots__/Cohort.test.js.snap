// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Person should match the snapshot with all data passed in correctly 1`] = `
<Fragment>
  <section
    className="people"
  >
    <Person
      id={1}
      key="1"
      name="Eric"
      photo="./eric.jpg"
      quote="You miss all the shots you don't take."
      superlative="Most Likely to Win the PGA"
    />
    <Person
      id={2}
      key="2"
      name="Amy"
      photo="./amy.jpg"
      quote="I believe in you, cuties!"
      superlative="Most Likely to Be an Actual Superhero"
    />
  </section>
</Fragment>
`;
